var/spack/repos/builtin/packages/dd4hep/package.py

```diff
@@ -55,13 +56,18 @@ def cmake_args(self):
             "-DCMAKE_CXX_STANDARD={0}".format(cxxstd),
             "-DDD4HEP_USE_XERCESC={0}".format(spec.satisfies('+xercesc')),
             "-DDD4HEP_USE_GEANT4={0}".format(spec.satisfies('+geant4')),
-            "-DBUILD_TESTING={0}".format(spec.satisfies('+testing')),
+            "-DDD4HEP_LOAD_ASSIMP={0}".format(spec.satisfies('+assimp')),
+            "-DBUILD_TESTING={0}".format(str(self.run_tests)),
```

Comment:
This `str(...)` conversion isn't actually needed is it?

Reply:
That's true, I removed it, thanks!